frontends/election-manager/src/screens/write_ins_screen.tsx

```diff
@@ -155,131 +78,17 @@ export function WriteInsScreen(): JSX.Element {
-        adjudication?.transcriptions.findIndex(
-          (t) => t.value === currenTranscription.transcribed_value
-        ) ?? -1;
-      if (!adjudication) {
-        // Add new adjudication and first transcription
-        newContestAdjudications[contestIndex].adjudications = [
-          ...newContestAdjudications[contestIndex].adjudications,
-          {
-            value: currenTranscription.adjudicated_value,
-            transcriptions: [
-              {
-                value: currenTranscription.transcribed_value,
-                count: 1,
-              },
-            ],
-          },
-        ].sort((a, b) => sortWithEmptyStringLast(a.value, b.value));
-      } else if (transcription && transcriptionIndex >= 0) {
-        // Add duplicate transcription
-        // eslint-disable-next-line no-plusplus
-        newContestAdjudications[contestIndex].adjudications[adjudicationIndex]
-          .transcriptions[transcriptionIndex].count++;
-      } else if (!transcription) {
-        // Add new transcription
-        const newAdjudicationTranscriptions = [
-          ...newContestAdjudications[contestIndex].adjudications[
-            adjudicationIndex
-          ].transcriptions,
-          {
-            value: currenTranscription.transcribed_value,
-            count: 1,
-          },
-        ].sort((a, b) => a.value.localeCompare(b.value));
-        newContestAdjudications[contestIndex].adjudications[
-          adjudicationIndex
-        ].transcriptions = newAdjudicationTranscriptions;
-        newContestAdjudications[contestIndex].adjudications = [
-          ...newContestAdjudications[contestIndex].adjudications,
-        ].sort((a, b) => sortWithEmptyStringLast(a.value, b.value));
-      }
-      return newContestAdjudications;
-    },
-    (contestsWithWritins || []).map((c) => ({
-      contestId: c.id,
-      adjudications: [
-        {
-          value: '',
-          count: 0,
-          transcriptions: [],
-        },
-      ],
-    }))
+  const contestsWithWriteIns = election.contests.filter(
+    (contest): contest is CandidateContest =>
+      contest.type === 'candidate' && contest.allowWriteIns
   );
-
-  function getAdjudicationsForContest(id: string): ContestAdjudicationRecord[] {
-    return (
-      contestAdjudications.find((a) => a.contestId === id)?.adjudications || []
-    );
-  }
+  const isPrimaryElection = contestsWithWriteIns.some((c) => c.partyId);
 
   function updateTranscriptions(
     transcribedValue: string,
     adjudicatedValue: string
   ) {
-    setTemporaryTranscriptions(
-      temporaryTranscriptions.map((t) => {
-        if (
-          t.transcribed_value === transcribedValue ||
-          t.transcribed_value === adjudicatedValue
-        ) {
-          return {
-            ...t,
-            adjudicated_value: adjudicatedValue,
-          };
-        }
-        return t;
-      })
-    );
+    console.log('updateTranscriptions', transcribedValue, adjudicatedValue);
```

Comment:
remove

Reply:
oh actually I'm assuming this is just temp until you implement this part so I guess its fine